tests/performance/assets/02-edgeconfigmap.yaml

```diff
@@ -54,6 +55,14 @@ data:
         docker-address: tcp://localhost:2375
         version: 2.0.0
```

Comment:
@edisonxiang should we maybe update the version field as well to match the current version?

Reply:
Thanks. done